src/Data/Weapons/Catalyst/EverlastingMoonglow/index.tsx

```diff
@@ -13,11 +13,11 @@ const key: WeaponKey = "EverlastingMoonglow"
 const data_gen = data_gen_json as WeaponData
 const hp_conv = [0.01, 0.015, 0.02, 0.025, 0.03]
 const [, trm] = trans("weapon", key)
-const normal_dmgInc = prod(subscript(input.weapon.refineIndex, hp_conv, { key: '_' }), input.premod.hp)
```

Comment:
There is no need for this change, right? The description here says "Normal Attack DMG is increased...", it shouldn't increase dmg for all elemental DMG.

Reply:
It's written that way to increase normal attack damage for all elements. Since we are worried about key bloat, and nothing in the game is specific to an element + move type combination, I'm just going to reduce the keys down. Then this will just turn back to what it was before